node/src/indexer/fake.rs

```diff
@@ -542,14 +591,15 @@ impl Drop for PausedIndexer {
         self.indexer_suspended.send(false).unwrap();
     }
 }
-
+#[derive(Clone, Copy, PartialEq, PartialOrd, Eq, Ord, From)]
+pub struct Uid(usize);
```

Comment:
Just guessing from the context. This is not a user ID right? Or what does the `U` stand for?

```suggestion
pub struct NodeId(usize);
```

Reply:
Uid stands for [unique identifier](https://en.wikipedia.org/wiki/Unique_identifier).
Until now, we used the `AccountId` as a unique identifier in the FakeIndexer. This won't work with the proposed disaster recovery plan.